www/include/Administration/parameters/api/help.php

```diff
@@ -12,9 +12,9 @@
     'help',
     'Enable Monitoring Engine Logs synchronisation between poller and Central Server operated by Centcore'
 );
-$help['tip_centcore_cmd_timeout'] = dgettext(
+$help['tip_gorgone_cmd_timeout'] = dgettext(
```

Comment:
does this option currently exist in gorgone ?

Reply:
Well actually Gorgone has lots of different timeouts on all his modules.
But this one here can/will be use when calling for an action like external commands.
So yes for now it's interesting to have a timeout set somewhere on the UI side.

The help sentence must be changed though.
Something like : "Timeout value in seconds. Used to make actions calls timeout."